lifetimes/estimation.py

```diff
@@ -278,6 +298,7 @@ def fit(self, frequency, monetary_value, iterative_fitting=4, initial_params=Non
                  This model is not very stable so we suggest >10 for best estimates evaluation.
```

Comment:
Depending on what you mean by stability, you may not need many iterations. At least on my dataset, removing monetary values of zero made a big difference to stability when measuring the MAE of the predictions. Before dropping zeros, multiple runs on the same data yielded highly variable results. Which dataset did you use and how did you measure stability?